.kokoro/gcp_windows_docker/Dockerfile

```diff
@@ -11,9 +13,12 @@ RUN powershell -Command $ProgressPreference = 'SilentlyContinue';
 ENV ChocolateyUseWindowsCompression false
 RUN powershell -NoProfile -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
 
-# Install .NET Core 3.1
+# Install .NET Core 3.0 which we need for Google.Apis.Auth.AspNetCore3
```

Comment:
It looks like we're still installing .NET Core 3.1 given the command?

Reply:
Ah yes, meant to change it bud didn't. Will push shortly and merge. Thanks!